src/base_object-inl.h

```diff
@@ -132,14 +132,14 @@ v8::EmbedderGraph::Node::Detachedness BaseObject::GetDetachedness() const {
 
 template <int Field>
 void BaseObject::InternalFieldGet(
-    v8::Local<v8::String> property,
+    v8::Local<v8::Name> property,
```

Comment:
It seems the variant with the Name parameters still have the wacky behavior of calling setters on get etc. And https://github.com/v8/node/pull/183 is a better fix here?

Reply:
Are you saying this change is wrong? I also took it from `v8/node`: https://github.com/v8/node/pull/174

Reply:
I can also add https://github.com/v8/node/pull/183 to this PR.

Reply:
From what I can tell the API used in https://github.com/v8/node/pull/183 is already available on main. https://github.com/v8/node/pull/174 migrates away from the old API to a new one though both have some strange behavior (calling setting on get etc.) while https://github.com/v8/node/pull/183 uses another API without the strange behavior. So it's not wrong though I figured we might as well just use the ultimate fix...

Reply:
Added https://github.com/v8/node/pull/183 with a reworded commit message.